docs/rules/no-console.md

```diff
@@ -49,6 +49,45 @@ console.error("Log an error level message.");
 
 If you're using Node.js, however, `console` is used to output information to the user and so is not strictly used for debugging purposes. If you are developing for Node.js then you most likely do not want this rule enabled.
 
+Another case where you might not use this rule is if you want to enforce console calls and not console overwrites. For example:
+
+```js
+console.error = function (message) {
+  throw new Error(message);
+};
+```
+
+With the `no-console` rule, this will receive a warning/error. For the above example, you can disable the rule:
```

Comment:
> With the `no-console` rule

=> With the `no-console` rule enabled? not sure, just checking.. (°°)～

Reply:
clarified with the example rule configuration so that an error would be reported